### DISCUSSION TITLE
Exporter composant react Explanation

### DISCUSSION THREAD

**Discussion by totakoko** [publicodes/publicodes#587] [Ideas]
Côté FCU, ça pourrait nous aider si on pouvait accéder au composant [Explanation](https://github.com/publicodes/publicodes/blob/bd5bca0f00180d7c5ce454e04547852b89a8953a/packages/react-
ui/src/Explanation.tsx#L61-L81) qui ne semble pas exporté pour le moment.

Le besoin est de pouvoir expliquer une valeur.

**Comment by johangirod** [comment]
Je n'y vois pas d'inconvénients :+1: 

Ça me semble être d'ailleurs une bonne direction à suivre pour le projet : exporter des composants de documentations qui puissent être composés par les réutilisateurs pour créer des pages d'explication sur mesure.

**Comment by johangirod** [comment]
Le composant est exporté dans les dernières versions de react-ui : 
https://publi.codes/docs/api/react-ui/functions/explanation
